src/components/OperatorChecklist.tsx

```diff
@@ -0,0 +1,289 @@
+    const { count, isLoading: isLoadingLiveNodes } = useOperatorLiveNodes(heartbeats)
+
+    const nodesRunning = isLoadingLiveNodes ? undefined : count > 0
+
+    useEffect(() => {
+        setNodesFunded(undefined)
+
+        if (!operator) {
+            return
+        }
+
+        if (!operator.nodes.length) {
+            /**
+             * No nodes = no funded nodes, eh? Let's report that
+             * and cut it short!
+             */
+            return void setNodesFunded(false)
+        }
+
+        let mounted = true
+
+        setTimeout(async () => {
+            class InsufficientFundsError {}
+
+            let result = false
+
+            try {
+                for (const nodeAddress of operator.nodes) {
+                    const balance = toBN(
+                        await getNativeTokenBalance(nodeAddress, defaultChainConfig.id),
+                    )
+
+                    if (!mounted) {
+                        return
+                    }
+
+                    if (balance.isLessThan(toAtto(0.1))) {
+                        throw new InsufficientFundsError()
+                    }
+                }
+
+                result = true
+            } catch (e) {
+                if (e instanceof InsufficientFundsError) {
+                    return
+                }
+
+                console.warn('Failed to check if nodes are funded', e)
+            } finally {
+                if (mounted) {
+                    setNodesFunded(result)
+                }
+            }
+        })
+
+        return () => {
+            mounted = false
+        }
+    }, [operator])
+
+    if (isLoading || isFetching) {
+        /**
+         * We're loading the operator. Make all checklist items undefined (all TBD).
+         */
+        return {}
+    }
+
+    if (!operator) {
+        /**
+         * We're done loading, and found no Operator. There are no checklist
+         * items to dash out.
+         */
+        return {
+            funded: false,
+            nodesDeclared: false,
+            nodesFunded: false,
+            nodesReachable: false,
+            nodesRunning: false,
+        }
+    }
+
+    const funded = operator.valueWithoutEarnings.isGreaterThan(0)
+
+    const nodesDeclared = operator.nodes.length > 0
+
+    return {
+        funded,
+        nodesDeclared,
+        nodesFunded,
+        nodesReachable,
+        nodesRunning,
+    }
+}
+
+function ChecklistItem({
+    children,
+    disabled = false,
+    state,
+    tip = '',
+}: {
+    children: ReactNode
+    disabled?: boolean
+    state?: boolean
+    tip?: ReactNode
+}) {
+    return (
+        <ChecklistItemRoot $disabled={disabled}>
+            <div>
+                {state === false && (
+                    <IconWrap $color={disabled ? void 0 : '#FF5C00'}>
+                        <JiraFailedBuildStatusIcon label="Error" size="medium" />
+                    </IconWrap>
+                )}
+                {typeof state === 'undefined' && (
```

Comment:
what about having 3 states - error/loading/success? it isn't so obvious that undefined state means that it's loading